### PR TITLE
PROD-71: Travis CI

### PR DESCRIPTION
**Pull request by sashamuliar**
# Reviewers

- [x] @vikabelous
- [x] @okohuch
- [x] @shevchukm
- [x] @imysheniuk

# Jira task
http://ssu-jira.softserveinc.com/browse/PROD-71

# Description
Have to configure Travis CI service. 
Have to create '.travis.yml' file. 
Have to install 'rubocop-ci' gem.
Have to configure Rspec on CI. 

# Checklist:

- [x] Rubocop doesn't raise any warnings and offenses
- [x] All specs pass
- [x] PR is approved by 2 team mates
- [x] PR is approved by 2 experts

### CODE REVIEW

**Review by shevchukm — APPROVED** [review]

**Review by imysheniuk — APPROVED** [review]

File: Gemfile [hunk 1]
```diff
@@ -12,6 +12,7 @@ gem 'devise', '~> 4.4.3'
 gem 'devise-i18n'
 gem 'haml', '~> 5.0.0'
 gem 'jbuilder', '~> 2.5'
+gem 'jquery-rails', '~> 4.3.1'
```
Comment (viktoriia-bilous) [inline Gemfile]: Do you need this gem for CI setup?

**Comment by viktoriia-bilous** [comment]
Why do you have two checks for the PR?
image

File: Gemfile (hunk 1 above)
Comment (sashamuliar) [reply Gemfile]: removed

**Comment by sashamuliar** [comment]
@vikabelous it's never too bad to have two checks passed
2 checks - 2 times safer )
leave check only for push

**Review by okohuch — APPROVED** [review]

**Review by viktoriia-bilous — APPROVED** [review]

**Pull request merged**